bitnami/pytorch/README.md

```diff
@@ -0,0 +1,161 @@
+| `livenessProbe.failureThreshold`       | Minimum consecutive failures for the probe to be considered failed after having succeeded.   | `5`                                                     |
+| `readinessProbe.enabled`               | Enable/disable the Readiness probe                                                           | `true`                                                  |
+| `readinessProbe.initialDelaySeconds`   | Delay before readiness probe is initiated                                                    | `5`                                                     |
+| `readinessProbe.periodSeconds`         | How often to perform the probe                                                               | `5`                                                     |
+| `readinessProbe.timeoutSeconds`        | When the probe times out                                                                     | `1`                                                     |
+| `readinessProbe.successThreshold`      | Minimum consecutive successes for the probe to be considered successful after having failed. | `1`                                                     |
+| `readinessProbe.failureThreshold`      | Minimum consecutive failures for the probe to be considered failed after having succeeded.   | `5`                                                     |
+| `persistence.enabled`                  | Use a PVC to persist data                                                                    | `true`                                                  |
+| `persistence.mountPath`                | Path to mount the volume at                                                                  | `/bitnami/pytorch`                                      |
+| `persistence.storageClass`             | Storage class of backing PVC                                                                 | `nil` (uses alpha storage class annotation)             |
+| `persistence.accessMode`               | Use volume as ReadOnly or ReadWrite                                                          | `ReadWriteOnce`                                         |
+| `persistence.size`                     | Size of data volume                                                                          | `8Gi`                                                   |
+| `persistence.annotations`              | Persistent Volume annotations                                                                | `{}`                                                    |
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```console
+$ helm install --name my-release \
+  --set pytorch.distributed.enabled=true \
+  --set pytorch.distributed.worldSize=8 \
+    bitnami/pytorch
+```
+
+The above command create 8 pods for PyTorch: one master and seven workers.
```

Comment:
Is there any limitation in the number of master and workers (e.g. maximimum workers, workers must be even, etc.)? In that case, you should add validations to ensure the user doesn't include a wrong number.

Reply:
I don't think so.